myning/tui/inventory.py

```diff
@@ -21,8 +23,8 @@ def on_click(self):
     def update(self):
         self.border_title = "Inventory"
         self.border_subtitle = (
-            f"{len(player.inventory.items)} items "
-            f"({Formatter.gold(sum(item.value for item in player.inventory.items))})"
+            f"{len(inventory.items)} items "
+            f"({Formatter.gold(sum(item.value for item in inventory.items))})"
```

Comment:
we can use `item.total_value` here. who knows, we might get lucky and these two lines could fit in one line 😂